ARKitExample/ViewController.swift

```diff
@@ -89,17 +89,19 @@ class ViewController: UIViewController {
         // As a TVIVideoCapturer, we must deliver CVPixelBuffers and not CGImages to the consumer.
         let pixelBuffer = self.copyPixelbufferFromCGImageProvider(image: imageRef!)
 
-        self.frame = TVIVideoFrame(timestamp: Int64((displayLink?.timestamp)! * 1000000),
-                                   buffer: pixelBuffer,
-                                   orientation: TVIVideoOrientation.up)
-        self.consumer?.consumeCapturedFrame(self.frame!)
+        if (pixelBuffer != nil) {
```

Comment:
We really should use if/let bindings here instead of doing a nil check and force unwrapping in the block. But it's something that we can address in the future.